src/mediaelement.js

```diff
@@ -135,6 +139,14 @@ export default class MediaElement extends WebAudio {
             media.load();
         }
 
+        // verify media element
+        if (
+            !(media instanceof HTMLMediaElement) ||
+            media.addEventListener === undefined
```

Comment:
i prefer typeof check for undefined
and should the verify be on top of the _load method? i am not sure

Reply:
done